package/pyproject.toml

```diff
@@ -12,3 +12,64 @@ requires = [
     "setuptools",
     "wheel",
 ]
+
+[project]
+name = "MDAnalysis"
+dynamic = ['version', 'readme']
```

Comment:
@tylerjereddy can I pick your brains on this?

Essentially adding a direct readme entry in the pyproject.toml is complicated because it accesses the README.rst which exists outside of the `package` directory. So I'm thinking we can possibly just leave it as `dynamic` since `setup.py` has set logic for gathering `long-description`, does that make sense to do?